New version: CameraCalibrations v0.9.5

JuliaRegistrator opened the pull request:
- Registering package: CameraCalibrations
- Repository: https://github.com/yakir12/CameraCalibrations.jl
- Created by: @github-actions[bot]
- Version: v0.9.5
- Commit: 866135ac3135133073e1bcc932e81698c98913c5
- Reviewed by: @github-actions[bot]
- Reference: https://github.com/yakir12/CameraCalibrations.jl/commit/866135ac3135133073e1bcc932e81698c98913c5#commitcomment-172382401
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6e1765409bc3ea5cb6d45d5d9ce526325181ff4d77ac809f1c234141ebee0304174a71c3657052cf47562ee8024e8a94ae858622602f8d235acfbc0a25201551f5a9f3f670950e6a3aaeaa400e700912cd67992a4b793d684ea8bf8e9ac1187bac7c32d17b9f22c8241ef06c55769f88d1d2df6ebef569c50c0d0eadcac2ec1c47c562f610c4454e147e0baa1b8f89489e8c9a9939635c0f946c39e9c2e272f20faa3b5f8b2ed4ec9efe891458991717b -->

yakir12 commented:
According to https://github.com/JuliaRegistries/General/blob/master/README.md#can-my-package-be-registered-if-it-requires-a-version-of-julia-that-is-not-yet-released, this can only be merged manually.

The issue is that libcxxwrap_julia_jll is held back by OpenCV related dependencies, and therefore CxxWrap doesn't pre-compile on Julia 1.12. Which is why I specified that only Julia v1.10 can be used with this package (CameraCalibrations).

Recent work in https://github.com/JuliaImages/OpenCV.jl/issues/61 is promising though... but in the mean time, merging this manually will help my specific use-case.

Thanks!

GunnarFarneback commented:
There is no Julia Compat change in this PR and your `Project.toml` says "1.10", which means 1.10 and higher 1.x versions.
Tip: [PackageCompatUI](https://github.com/GunnarFarneback/PackageCompatUI.jl) is helpful at getting the compat specifications right.

yakir12 commented:
Sorry, my mistake, I've corrected it (only) now.

yakir12 commented:
OK! So now, unless I'm wrong again, this needs a **manual merge** (only because `AutoMerge` doesn't run with the Julia versions specified in `Compat`).

JuliaRegistrator committed:
New version: CameraCalibrations v0.9.5

UUID: 70c3cbac-2c23-4b2d-966c-5a25c0a540fb
Repo: https://github.com/yakir12/CameraCalibrations.jl.git
Tree: d998676ec920f08b73131df36c555f6e56bce814

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
